cms/djangoapps/contentstore/tasks.py

```diff
@@ -737,14 +739,18 @@ def validate_course_olx(courselike_key, course_dir, status):
     if not course_import_olx_validation_is_enabled():
         return olx_is_valid
     try:
+        if str(courselike_key) == "course-v1:ArbiX+CS101+2014_T3":
+            logging.info(log_prefix + "Validating. Calling olxcleaner.validate")
```

Comment:
what is `log_prefix` here?

Reply:
This is the default prefix they are using in every log of this function. This string is equal to f"Course import {courselike_key}"